report/launch_local_fi.sh

```diff
@@ -15,12 +15,18 @@
 
 BENCHMARK_SET=$1
 
+. /fi/tools/web-fuzzing-introspection/.venv/bin/activate
+
 # Create (small) DB.
 (cd /fi/tools/web-fuzzing-introspection/app/static/assets/db/ && \
     find "/experiment/benchmark-sets/$BENCHMARK_SET" -type f -name "*.yaml" \
     -exec basename {} .yaml \; | sort > must_include_small.config && \
     ./launch_minor_oss_fuzz.sh)
 
+/fi/tools/web-fuzzing-introspection/.venv/bin/deactivate
+
+touch /FI_DB_READY
```

Comment:
@DavidKorczynski What would be the best way to check if local FI is ready to use?
I am using this file as a naive way to mark the completion of DB creation so that another process knows when to resume.
If there is a better way, I would love to learn.

Thanks!

Reply:
Hmm, the easiest may be to ping it after running `main.py`? The following shows connection refused while loading to providing the html when the app is live.

```
(.venv) dav@ubuntu-main:/tmp$ wget http://localhost:8080/
--2024-03-23 14:05:04--  http://localhost:8080/
Resolving localhost (localhost)... 127.0.0.1
Connecting to localhost (localhost)|127.0.0.1|:8080... failed: Connection refused.

(.venv) dav@ubuntu-main:/tmp$ wget http://localhost:8080/
--2024-03-23 14:05:05--  http://localhost:8080/
Resolving localhost (localhost)... 127.0.0.1
Connecting to localhost (localhost)|127.0.0.1|:8080... failed: Connection refused.

(.venv) dav@ubuntu-main:/tmp$ wget http://localhost:8080/
--2024-03-23 14:05:06--  http://localhost:8080/
Resolving localhost (localhost)... 127.0.0.1
Connecting to localhost (localhost)|127.0.0.1|:8080... failed: Connection refused.

(.venv) dav@ubuntu-main:/tmp$ wget http://localhost:8080/
--2024-03-23 14:05:07--  http://localhost:8080/
Resolving localhost (localhost)... 127.0.0.1
Connecting to localhost (localhost)|127.0.0.1|:8080... connected.
HTTP request sent, awaiting response... 200 OK
Length: 75079 (73K) [text/html]
Saving to: ‘index.html’
index.html                                                                               100%[===============================================================================================================================================================================================================================>]  73.32K  --.-KB/s    in 0s      
2024-03-23 14:05:07 (428 MB/s) - ‘index.html’ saved [75079/75079]




```

Reply:
Thanks! I will try this.